.github/workflows/tests.yml

```diff
@@ -47,8 +47,15 @@ jobs:
       - name: Setup Memcached
         uses: niden/actions-memcached@v7
 
+      - name: Setup problem matchers
+        run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
       - name: Install dependencies
-        run: composer update --${{ matrix.stability }} --prefer-dist --no-interaction --no-progress
+        uses: nick-invision/retry@v1
+        with:
+          timeout_minutes: 5
+          max_attempts: 5
```

Comment:
Can we get rid off these timeout and attempts in the PR? Don't think we really need those?

Reply:
They are good to stop random failures due to networking. This sometimes is the cause of failing PR tests.

Reply:
Okay, we can keep them. No worries.